app/models/content_item.rb

```diff
@@ -46,7 +46,7 @@ def self.create_or_replace(base_path, attributes)
   field :_id, :as => :base_path, :type => String, :overwrite => true
   field :content_id, :type => String
   field :title, :type => String
-  field :description, :type => String
+  field :description, :type => Hash, :default => { "value" => nil }
```

Comment:
Yes, but where previously calling `description` on a new object would've resulted in a blank string, does it not now result in a `nil`?